assets/readme.md

```diff
@@ -1,19 +1,121 @@
+
+You can find your key in your Clearbit dashboard under the API menu link or by navigating to [https://dashboard.clearbit.com/api](https://dashboard.clearbit.com/api). Clearbit provides two keys, a *secret API key* and a *publishable API key*. Use the **secret API key** to integrate Hull with Clearbit:
+![Find your Clearbit API Key](./docs/clearbitapi01.png)
+
+## Enrichment
+
+The Enrichment feature of the connector allows you to complete a users profile with more than 85 data points provided by Clearbit’s API.
+
+**Configure Enrichment**
+You can enable or disable this feature by toggling the checkbox in the **Enrichment** section of the Settings tab:
+![Enrichment 1](./docs/enrichment01.png)
+
+Enrichment is enabled by default but will only result in API calls to Clearbit’s API if you specify at least one segment in the provided field. You can search segments by simply typing a part of the name into the field or by opening the dropdown via the triangular button.
+![Enrichment 2](./docs/enrichment02.png)
+
+**Which users do get enriched?**
+The connector only enriches users who have an *email address* *and have never been enriched before*. The connector sets the trait `clearbit/enriched_at` with the value of the timestamp if the user has been sent to Clearbit’s API.
+
+![Enrichment 3](./docs/enrichment03.png)
+
+Note: The connector will perform the enrich operation exactly only once, even if this API call does not yield additional data points.
+
+**When do users get enriched?**
+The connector attempts the enrichment as soon as the user enters one of the whitelisted segments. Please note that the segment needs to exist before the user gets created, otherwise this automatic enrichment will not be performed. 
+You can always enforce the enrichment by selecting one or multiple users on the users page and send them to your Clearbit connector via the “Actions” button. If you perform this route, the whitelisted segments will be ignored and all users that have an email address and no trait `clearbit/enriched_at` will be processed.
+
+Please note that users get only enriched ***exactly once***. Even if this enrichment does not yield any data, the Connector will not run another enrichment.
+
+**How long does it take to enrich users?**
+In most cases this process will take no longer than a couple of minutes. In some cases the Clearbit API marks a lookup as pending, which results in a delay of processing for up to an hour.
+
+## Reveal
+
+The Reveal feature of the connector allows you to turn your anonymous website traffic into company profiles to power targeted campaigns and close more named accounts .
+
+You can enable or disable the feature by toggling the checkbox in the **Reveal** section of the Settings tab:
+![Reveal 1](./docs/reveal01.png)
+
+Reveal will turn anonymous users into Company profiles. By default, your Hull organization does not store data about anonymous traffic, so you need to enable “Anonymous Users” for your organization.
```

Comment:
@unity and @sbellity, question to clarify this: If I enable "Anonymous Users" at a later point of time, I would have all the data to run Reveal? Yes or No? 
If **yes**, I would suggest to change the wording and make it "By default, your Hull organization does not handle data from anonymous traffic, [...]"
If no, I really don't see the necessity to change the wording, because again perceived reality is that it doesn't work, so I as the end user don't care about technical details, whether the data is stored or not, the data is useless to me in this scenario.